common/services/prismic/pages.js

```diff
@@ -48,6 +48,7 @@ export function parsePage(document: PrismicDocument): Page {
   return {
     type: 'pages',
     ...genericFields,
+    showOnThisPage: data.showOnThisPage || false,
```

Comment:
found this a bit confusing - my immediate question was," show what on this page? but with it now - and not sure I can think of a better name for OnThisPage